package/utils/__tests__/objectify.js

```diff
@@ -1,4 +1,5 @@
 /* global test expect */
+console.log('objectify test cwd', process.cwd());
```

Comment:
Is this intentional?

Reply:
No, that was a debugging line that should be removed (and looks like you've done that on master already).